src/planner/plan_util.cpp

```diff
@@ -0,0 +1,100 @@
+//===----------------------------------------------------------------------===//
+//
+//                         Peloton
+//
+// plan_util.cpp
+//
+// Identification: src/planner/plan_util.cpp
+//
+// Copyright (c) 2015-18, Carnegie Mellon University Database Group
```

Comment:
Fix copyright.

Reply:
Change what? Write it as 2018 instead of 18? We should also fix it in the formatter.py script so that it complains for mistakes like this.

Reply:
And FWIW, the same pattern, eg: 2015-17 is being followed in many other files as well. See create_plan.cpp , eg:.

Reply:
Correct, years should be full. In general, if you run the formatter script [here](https://github.com/cmu-db/peloton/blob/master/script/formatting/formatter.py) to add the header, it should do it correctly.

You're correct that other files don't adhere to the rules. This is something we're fixing incrementally rather than submitting a giant PR that breaks other ongoing PRs.

Reply:
cool; pushed the new changes. Yeah, turns out I missed the `-u` flag to the script.